### ISSUE TITLE
Conditional fields in Block context

### ISSUE THREAD

**Issue by jan-dh** [verbb/vizy#233]
### What are you trying to do?

I would love to have the ability to have, just like in an entry, the options to show/hide fields in a block, based on for example a lightswitch field in that same field. Currently the conditional options are only available on user level. 

### What's your proposed solution?

Expose extra conditional options.

**Comment by engram-design** [comment]
Yep, it’s been mentioned before https://github.com/verbb/vizy/issues/64 but we’d have to write all the conditional logic in Vue as we can’t use Craft’s own conditional field handling
